lib/jara/releaser.rb

```diff
@@ -111,26 +113,19 @@ def git_remote
       @git_remote ||= @shell.exec('git config --get remote.origin.url')
     end
 
-    def jruby_version
-      @jruby_version ||= begin
-        jruby_jars_path = $LOAD_PATH.grep(/\/jruby-jars/).first
-        jruby_jars_path && jruby_jars_path.scan(/\/jruby-jars-(.+)\//).flatten.first
-      end
-    end
-
     def metadata
       m = {
         'packaged_by' => "#{ENV['USER']}@#{Socket.gethostname}",
         'sha' => branch_sha,
         'remote' => git_remote,
-        'jruby' => jruby_version,
       }
       m.merge!(@extra_metadata)
+      m.merge!(@archiver.metadata)
```

Comment:
Technically, I'm guessing that the order should be flipped here. A custom archiver should not be allowed to override the above properties (I guess)